yt/python/yt/environment/yt_env.py

```diff
@@ -676,9 +676,11 @@ def start(self, on_sequoia_started_func=None, on_masters_started_func=None):
             self._write_environment_info_to_file()
             logger.info("Environment started")
         except (YtError, KeyboardInterrupt) as err:
+            self.pids_file.close()
             logger.exception("Failed to start environment")
             self.stop(force=True)
             raise YtError("Failed to start environment", inner_errors=[err])
+        self.pids_file.close()
```

Comment:
It looks incorrect, since there are tests that run some components with direct calls of methods `run_*`

In general scenario this file should be closed in `stop` method